### PR TITLE
Throw an error when trying to load a scene in "play" mode that in not…

### PR DESCRIPTION
**Pull request by ghost**
## Purpose of this PR
Throw an error when trying to load a scene in "play" mode that in not in the plugin configuration

## Testing status
* No tests have been added.

### CODE REVIEW

File: com.stansassets.scene-management/BuildConfigurator/Runtime/BuildConfiguration.cs [hunk 1]
```diff
@@ -178,5 +179,11 @@ BuildTargetRuntime ConvertRuntimePlatformToBuildTarget(RuntimePlatform platform)
                     throw new ArgumentOutOfRangeException(nameof(platform), platform, null);
             }
         }
+
+        public bool HasScene(string sceneName)
```
Comment (stan-osipov) [inline com.stansassets.scene-management/BuildConfigurator/Runtime/BuildConfiguration.cs:183]: why is it public?

File: com.stansassets.scene-management/Runtime/Statis/AdditiveScenesLoader.cs [hunk 2]
```diff
@@ -459,14 +459,7 @@ static void ProcessSceneUnLoad(string sceneName)
...
         {
-            // TODO let's reconsider the validation
-            /*
-            if (!Application.isEditor && BuildConfigurationSettings.Instance.HasValidConfiguration)
-            {
-                return BuildConfigurationSettings.Instance.Configuration.HasScene(sceneName);
-            }*/
-
-            return true;
+            return BuildConfigurationSettings.Instance.Configuration.HasScene(sceneName);
```
Comment (stan-osipov) [inline com.stansassets.scene-management/Runtime/Statis/AdditiveScenesLoader.cs:462]: Looks like `BuildConfigurationSettings.Instance.HasValidConfiguration` was there for a reason. What if we just installed the plugin and we don't have a configuration yet?

**Review by stan-osipov — CHANGES_REQUESTED** [review]
some quastions

**Comment by yurii-lunha-stansassets** [comment]
Recreated with right conventions

**Pull request closed without merging**